rdr_service/tools/tool_libs/biobank_orders.py

```diff
@@ -18,12 +21,23 @@ def run(self):
         if self.args.command == 'send-to-mayolink':
             self.upload_order_to_mayolink(barcode=self.args.barcode)
 
+    def _load_mail_kit(self, barcode: str, session: Session) -> Optional[BiobankMailKitOrder]:
+        mail_kit_orders = MailKitOrderDao.get_with_barcode(barcode=barcode, session=session)
+        if len(mail_kit_orders) == 0:
```

Comment:
:nit Since you are returning a list, you could say `if not mail_kit_orders:`

Reply:
👍 I can't figure out where I land on that. I like the explicit check in some scenarios, but `if not mail_kit_orders` does feel like it fits well here.